tests/byte_buf_test.c

```diff
@@ -161,7 +161,6 @@ static int s_test_buffer_eq_fn(struct aws_allocator *allocator, void *ctx) {
     struct aws_byte_buf b2 = aws_byte_buf_from_c_str("testb");
 
     b1.capacity = 5;
-    b1_equal.capacity = 2;
```

Comment:
why did you remove this?

Reply:
this assignment results on an invalid `aws_byte_buf` structure, since its `.capacity` would be smaller than its actual `.len`.